CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Replace OuiSelect component with OuiSuperSelect in data-source plugin ([#5315](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5315))
 - [Workspace] Add core workspace service module to enable the implementation of workspace features within OSD plugins ([#5092](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5092))
 - [Workspace] Setup workspace skeleton and implement basic CRUD API ([#5075](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5075))
+- [Workspace] Add ACL related functions ([#5084](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5084/))
```

Comment:
Is it possible to create a short video and add it to this PR, so everyone can visually see the permission control behavior?

Reply:
I'm glad to do that, but for this PR, it only works when the caller triggers the functions call, and the code of the caller will be in the following PR, so it's hard to tigger these basic functions in this PR by UI now. We will create a video for permission control in the following PR later.